ical.NET/Serialization/iCalendar/Serializers/DataTypes/GeographicLocationSerializer.cs

```diff
@@ -45,9 +46,9 @@ public GeographicLocation Deserialize(string value)
 
             double lat;
             double lon;
-            double.TryParse(values[0], out lat);
-            double.TryParse(values[1], out lon);
-            g.Latitude = lat;
+			double.TryParse(values[0], NumberStyles.Any, CultureInfo.InvariantCulture, out lat);
```

Comment:
You have some weird indentation here. Can you line it up with the rest of the lines?